### DISCUSSION TITLE
Settings menu

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#269] [Q&A] [answered]
In my opinion, the settings menu should be thoroughly revised.
One searches forever!
E.g.: The things that are shown in the display can be found under "Navigation".
there needs to be a dedicated button.
The "Profile" button, on the other hand, could be under "Routes".
Under "Routes" is the BRouter(offline) explicitly. The others under "Options"??
I think everyone should think about this and make suggestions for improvement to make the whole thing a little clearer..

**Comment by 0709wiwiwi** [comment]
Menu simplicity and many new additional features ?
Could the following possibly be helpful in this regard ?
https://github.com/devemux86/cruiser/discussions/64

**Comment by devemux86** [comment, marked as answer]
Agreed that improvements are always welcome! :slightly_smiling_face: 

> The things that are shown in the display can be found under "Navigation".
there needs to be a dedicated button.

All options in "Navigation | Display" are for panels that appear in navigation.
(only 2-3 may also appear in follow mode)

> The "Profile" button, on the other hand, could be under "Routes".

Profile has global settings for route planning, navigation, voice guidance, etc.
It does not belong under a specific category.

e.g. OsmAnd has the profiles in the most prominent place.

> Under "Routes" is the BRouter(offline) explicitly. The others under "Options"??

This was after a suggestion in the past. Isn't BRouter (offline) the most popular router in Cruiser?
I can move it under Options, but will involve more steps to find it.

Generally we should avoid too many categories in the main settings screen (it may be confusing).

**Comment by devemux86** [comment]
> Menu simplicity and many new additional features ?

These two can't really co-exist! :slightly_smiling_face: 

For example, with navigation / voice guidance or the new nearby POI and favorites.
I can select myself (like other apps do!) fixed panels, fixed distances, fixed options.

Far fewer settings, but then how do we all expect Cruiser to work to your liking? :slightly_smiling_face:

**Reply by 0709wiwiwi** [reply]
> I can select myself (like other apps do!) fixed panels, fixed distances, fixed options.

Cruiser default Preset multi choice at the initial app start ?
These Cruiser defaults can later be fine tuned by users
https://github.com/devemux86/cruiser/discussions/165
Key Problem: 
A generally useful Preset per transport type has to be determined.
